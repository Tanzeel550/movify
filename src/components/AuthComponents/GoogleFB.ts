import React from 'react';
import { startLoginWithGoogle } from '../../actions/authActions';

const GoogleFB = () => (
  <div className="flex-c-m">
    <button className="social__item facebook-icon">
      <i className="zmdi zmdi-facebook" />
    </button>

    <button className="social__item google-icon" onClick={startLoginWithGoogle}>
      <i className="zmdi zmdi-google" />
    </button>
  </div>
);

export default GoogleFB;
