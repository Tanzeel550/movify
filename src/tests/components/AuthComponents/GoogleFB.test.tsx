import { shallow, ShallowWrapper } from 'enzyme';
import GoogleFB from '../../../components/AuthComponents/GoogleFB';
import React from 'react';

test('should render GoogleFB component correctly', () => {
  const wrapper: ShallowWrapper<
    any,
    Readonly<{}>,
    React.Component<{}, {}, any>
  > = shallow(<GoogleFB />);
  expect(wrapper).toMatchSnapshot();
});

export {};
