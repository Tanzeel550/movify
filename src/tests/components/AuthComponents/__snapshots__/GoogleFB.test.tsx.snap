// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render GoogleFB component correctly 1`] = `
<div
  className="flex-c-m"
>
  <button
    className="social__item facebook-icon"
  >
    <i
      className="zmdi zmdi-facebook"
    />
  </button>
  <button
    className="social__item google-icon"
    onClick={[Function]}
  >
    <i
      className="zmdi zmdi-google"
    />
  </button>
</div>
`;
